test/e2e_flags.go

```diff
@@ -49,6 +49,10 @@ type ServingEnvironmentFlags struct {
 	ExceedingMemoryLimitSize int    // Memory size used to trigger a non-200 response when the service is set with 300MB memory limit.
 	RequestHeaders           string // Extra HTTP request headers sent to the testing deployed KServices.
 	IngressClass             string // Ingress class used for serving.
+	CustomMemoryRequests     int    // Memory requests used for services with a specific size.
+	CustomMemoryLimits       int    // Memory limits used for services with a specific size.
+	CustomCPURequests        int    // CPU requests used for services with a specific size.
+	CustomCPULimits          int    // CPU limits used for services with a specific size.
```

Comment:
Can we make these strings like the k8s API does? Changing them to arbitrary share units feels incorrect.

Reply:
I was following the convention set by `ExceedingMemoryLimitSize`, but I can give that a try.

Reply:
You should be able to [use `resource.MustParse`](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#MustParse) to interpret these, and then [call `MilliValue()`](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity.MilliValue) to extract the value.